### PR TITLE
Add source info for contributing Doc

### PR DESCRIPTION
**Pull request by sthirugn**

### CODE REVIEW

File: moduleroot/CONTRIBUTING.md.erb [hunk 1]
```diff
@@ -1,3 +1,13 @@
+Source for this document
+========================
+
+- This document is part of
```
Comment (ekohl) [inline moduleroot/CONTRIBUTING.md.erb]:
```suggestion
- This document is shared between
```

File: moduleroot/CONTRIBUTING.md.erb [hunk 2]
```diff
@@ -1,3 +1,13 @@
+Source for this document
```
Comment (ekohl) [inline moduleroot/CONTRIBUTING.md.erb]: This should be at the very bottom IMHO. It's the least relevant part of the document for someone who wants to contribute to the module itself.

File: moduleroot/CONTRIBUTING.md.erb (hunk 1 above)
Comment (sthirugn) [reply moduleroot/CONTRIBUTING.md.erb]: :+1:

File: moduleroot/CONTRIBUTING.md.erb (hunk 2 above)
Comment (sthirugn) [reply moduleroot/CONTRIBUTING.md.erb]: :+1:

**Comment by sthirugn** [comment]
@ekohl thank you for the review, comments updated.

**Comment by sthirugn** [comment]
> I was thinking a bit more about this and wonder if we shouldn't explain that [modulesync](https://github.com/voxpupuli/modulesync) is used with https://github.com/theforeman/foreman-installer-modulesync as the config repository to sync various files, including `CONTRIBUTING.md`. How about something like:
> 
> ```
> Modulesync
> =========
> 
> Various files, including this one, are [modulesynced](https://github.com/voxpupuli/modulesync) using https://github.com/theforeman/foreman-installer-modulesync as its configuration. Changes should be made over there and then synced to all [managed modules](https://github.com/theforeman/foreman-installer-modulesync/blob/master/managed_modules.yml).
> ```

I like this and updated now.

**Pull request merged**